sofascoreAcademy/sofascoreAcademy/ViewModel/EventsViewModel.swift

```diff
@@ -48,5 +50,23 @@ class EventsViewModel: ObservableObject {
             return LeagueSection(league: league, matches: events.sorted { $0.startTimestamp < $1.startTimestamp })
         }
     }
+    
+    func insertIntoDB(_ sections: [LeagueSection]) {
+        sections.forEach { section in
+            let league = section.league
+            let leagueEntity = DBLeague(from: league)
+            try? DbManager.shared.dbQueue?.write { db in
+                try? leagueEntity.upsert(db)
+            }
+            
+            let events = section.matches
+            events.forEach { event in
+                let eventEntity = DBEvent(from: event)
+                try? DbManager.shared.dbQueue?.write { db in
+                    try? eventEntity.upsert(db)
+                }
+            }
+        }
+    }
```

Comment:
Ovo isto će ići preko StorageManagera. Taj će ti biti prvi layer apstrakcije nad data layerom, što se zove još Repository pattern.